pom.xml

```
@@ -271,6 +271,23 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>pw.krejci</groupId>
+                <artifactId>jmh-maven-plugin</artifactId>
+                <version>0.2.2</version>
+                <executions>
+                    <execution>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>jmh</goal>
```

```suggestion
                            <goal>benchmark</goal>
```

But I am not sure if we want to run it on every test

---

Updated to not run during `test`s. One can still run `./mvnw jmh:benchmark` to manually execute benchmarks.